recipes/libpng/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.6.21" %}
```

Comment:
The default channel version is `1.6.17`, but the ftp no longer has that version. However, it seems that both are `libpng16`, so we should be OK here.